.github/workflows/cd.yml

```diff
@@ -68,3 +68,12 @@ jobs:
         tags: ${{ steps.build.outputs.tags }}
         username: ${{ secrets.QUAY_USERNAME }}
         password: ${{ secrets.QUAY_ROBOT_TOKEN }}
+
+    - name: Push aio to quay.io
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        registry: "quay.io/${{ env.quay_org }}"
+        image: aio
```

Comment:
do we want this to be called something like fdo-aio?

Reply:
last time with patrick we said fdo was a repetition - perhaps `aio-server`?

Reply:
if the org is fdo I think just aio is fine